config/locales/hyrax.en.yml

```diff
@@ -595,13 +595,15 @@ en:
           add_to_collection:        "Add to collection"
           admin_set_confirmation:  "Deleting an admin set from %{application_name} is permanent. Click OK to delete this admin set from %{application_name}, or Cancel to cancel this operation"
           collections_confirmation_html: "<span class='pluralized'>Deleting this collection</span> will permanently remove the collection from the repository.  Items in the collection will remain in the repository.  Are you sure you want to delete this collection?"
+          collections_confirmation_no_items_html: "Are you sure you want to delete this collection? This action cannot be undone."
           delete_admin_set:        "Delete administrative set"
           delete_collection:       "Delete collection"
           delete_collections:      "Delete collections"
-          delete_collections_deny: "You must select one or more collections to delete."
```

Comment:
* Dashboard -> Collections
* click Delete Collections (bulk delete button on left)

Notice that the "You do not have sufficient privileges to edit this document" is displayed instead of the original "You must select one or more collections to delete."

Reply:
I was using an existing modal, when I should have created a new one. Good catch.